config/main.py

```diff
@@ -295,6 +295,10 @@ def _stop_services():
         'hostcfgd',
     ]
 
+    # on Mellanox platform pmon is stopped by syncd
+    if (version_info and version_info.get('asic_type') == 'mellanox'):
```

Comment:
Parenthesis not required here (lets remove to be more python-way consistent).
Also, in which cases ```bool(version_info) == False``` ? Let's teat it as undefined platform error, otherwise we hide the problem and keep pmon in the list

Reply:
At the beginning of the file lets initialize asic_type like this:

```
try:
    asic_type = version_info['asic_type']
except KeyError, TypeError:
    click.abort('undefined ASIC type')
```

Any subsequent code that uses asic_type knows that it is initialized correctly